app/models/concerns/fog_extensions.rb

```diff
@@ -51,6 +51,7 @@ module FogExtensions
 
 if Foreman::Model::Vmware.available?
   require 'fog/vsphere'
+  require 'fog/vsphere/models/compute/network'
```

Comment:
Could we add
```ruby
require 'fog/vsphere/models/cluster'
```
to solve the https://community.theforeman.org/t/foreman-1-20-2-vmware-sporadic-http-500-when-querying-api-compute-resources-api/13907?